tests/auto/tst_declarativetabmodel/tst_declarativetabmodel.cpp

```diff
@@ -283,9 +281,6 @@ void tst_declarativetabmodel::multipleTabsWithSameUrls()
     QCOMPARE(tabModel->data(modelIndex, DeclarativeTabModel::UrlRole).toString(), page2Tab1Url);
     QVERIFY(tabModel->data(modelIndex, DeclarativeTabModel::TitleRole).toString().isEmpty());
 
-    tabModel->updateTitle(tab2, true, page2Tab2Url, page2Tab2Title);
-    QCOMPARE(tabModel->activeTab().title(), page2Tab2Title);
-
```

Comment:
Would be good to test that both url and title are updated correctly. One approach that we could take is that we utilize the mock webpage, add apis to the mock for setting url and title, and connect the mock the model. This way test case could control input and verify that model updates correctly.